### ISSUE TITLE
Add /bin path when using --system-site-packages

### ISSUE THREAD

**Issue by psomhorst** [yyuu/pyenv-virtualenv#142]
When using system site packages linked to a version installed by pyenv, no link to the 'original' version's /bin path is added. Due to this, pip-installed executables like `ipython` do not work.

For example, I have a virtualenv created based on python 3.4.3, installed with pyenv.

    env PYTHON_CONFIGURE_OPTS="--enable-framework CC=clang" pyenv virtualenv --system-site-packages 3.4.3 my_virtual_env

Everything works fine from within python, but I can't run `ipython`.

Is it possible to add the `versions/3.4.3/bin` path of the python 3.4.3 version to the $PATH? Or would that mess up and actually run ipython with the python 3.4.3 version?

**Comment by psomhorst** [comment]
I tested whether running the `ipython` executable from the `3.4.3/bin` directory would result in using the 3.4.3 version, or the virtual version of python.

    pyenv activate my_virtual_env
    pip install radio
    ~/.pyenv/versions/3.4.3/bin/ipython
    In [1]: import radio
    In [2]: radio
    Out[2]: <module 'radio' from '/Users/########/.pyenv/versions/3.4.3/envs/my_virtual_env/lib/python3.4/site-packages/radio/__init__.py'>

    pyenv deactivate
    pyenv shell 3.4.3
    ipython
    In [1]: import radio
    ---------------------------------------------------------------------------
    ImportError                               Traceback (most recent call last)
    <ipython-input-1-fc8dbe8a75a8> in <module>()
    ----> 1 import radio

    ImportError: No module named 'radio'

In other words: I think it would be no problem (at least for the `ipython` executable) to just add the 3.4.3/bin directory to the PATH in order to be able to use the pip-installed executables.

**Comment by yyuu** [comment]
There's https://github.com/yyuu/pyenv-virtualenv/blob/51ebc8ff8a962de28d7f4fa661b981dc38f4d162/etc/pyenv.d/which/system-site-packages.bash to look up executables from its source version if current version is a virtualenv with `--system-site-packages`. Because of pyenv's shims mechanism, basically I don't like to add manage PATH in it. pip installed `ipython` is just working for me.
